core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -1924,7 +1924,9 @@ private long loadMetadataIfNotExistAndJournal(LockedInodePath inodePath,
       try {
         return loadMetadataAndJournal(inodePath, options);
       } catch (Exception e) {
-        LOG.error("Failed to load metadata for path: {}", inodePath.getUri());
+        // NOTE, this may be expected when client tries to get info (e.g. exisits()) for a file
+        // existing neither in Alluxio nor UFS.
+        LOG.debug("Failed to load metadata for path from UFS: {}", inodePath.getUri());
```

Comment:
@apc999  After the recent fixes (e.g. stop loading metadata for non-persisted files), will this still happen?